.github/labels.yml

```diff
@@ -0,0 +1,211 @@
+###
+### Special magic GitHub labels
+### https://help.github.com/en/github/building-a-strong-community/encouraging-helpful-contributions-to-your-project-with-labels
+#
+- name: "good first issue"
+  color: 7057ff
+  description: "Good for newcomers"
+- name: "help wanted"
+  color: 008672
+  description: "Extra attention is needed"
+
+###
+### Areas
```

Comment:
I'd say:
* chain/vm
* chain/sync
* chain[/misc]
* sealing/fsm
* sealing/storage
* proving
* mining
* client/storage
* client/retrieval
* wallet
* paych
* multisig
* networking

(looks like `description:`s aren't matching)

Reply:
Thanks! This is what I needed!

Reply:
done